staging/src/k8s.io/component-base/logs/options.go

```diff
@@ -70,11 +70,29 @@ func (o *Options) Validate() []error {
 	return errs
 }
 
-func flagIsSet(name string) bool {
+// Normalize replaces underscores with hyphens
+// we should always use hyphens instead of underscores when registering component flags
+func Normalize(s string) string {
+	return strings.Replace(s, "_", "-", -1)
+}
+
+// UnNormalize replaces hyphens with underscores
+// we should always use underscores instead of hyphens when validate flags
+func UnNormalize(s string) string {
+	return strings.Replace(s, "-", "_", -1)
+}
```

Comment:
pflag flagsets already have normalization functions (`GetNormalizeFunc()`)... we should use those instead of defining unrelated ones here